### PR TITLE
[onert] Avoid preparing cache for ruy kernel on fc layer, cpu

### PR DESCRIPTION
**Pull request by YongseopKim**
Avoid preparing cache for ruy kernel of fc layer on cpu.
The only fc hybrid will use ruy kernel to cache.

ONE-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc [hunk 1]
```diff
@@ -164,6 +164,13 @@ void FullyConnectedLayer::run()
 void FullyConnectedLayer::prepare()
 {
 #ifdef USE_RUY_GEMV
+  // The only fc hybrid will use ruy kernel
```
Comment (wateret) [inline runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:167]: What about leaving a TODO comment that this is a workaround and it should check if the weight is const.

Comment (ragmani) [reply runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:167]: It should also check if the input is dynamic.

**Comment by YongseopKim** [comment]
Sorry for late. Could you take a look please? @wateret and @ragmani

File: runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc [hunk 2]
```diff
@@ -164,13 +164,17 @@ void FullyConnectedLayer::run()
...
+  // TODO This is workaround
   // The only fc hybrid will use ruy kernel
   if (_input->data_type() != OperandType::FLOAT32 ||
       _weights->data_type() != OperandType::QUANT_INT8_SYMM)
   {
     return;
   }
 
+  // For now, dynamic input cannot support
+  assert(_input->is_dynamic() == false);
```
Comment (ragmani) [inline runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:176]: Why did you add this as an assertion?
If `_input` is dynamic, this method is not called?

Comment (YongseopKim) [reply runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:176]: Is this your comment's intention? I understood that this fc doesn't support dynamic tensor as input. Exactly what does it 
mean?

> It should also check if the input is dynamic.

Comment (YongseopKim) [reply runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:176]: FYI I'm not good at dynamic tensor code.

Comment (ragmani) [reply runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc:176]: If so, could you add a TODO comment instead of the assertion.

**Comment by YongseopKim** [comment]
PTAL
